apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -107,7 +106,6 @@ const USE_CASES = [
   GetNovuProviderCredentials,
   SelectIntegration,
   SelectVariant,
-  GetSubscriberPreference,
```

Comment:
I don't see this usecase being used anywhere in `worker`. 

It wouldn't start if some used usecase would be missing.

Reply:
i believe it should be fine, the use cases that are used in Worker to execute preferences are GetSubscriberTemplatePreference and static method GetPreferences.mapWorkflowPreferencesToChannelPreferences